api/v1beta1/ovndbcluster_types.go

```diff
@@ -200,5 +200,8 @@ func (instance OVNDBCluster) GetExternalEndpoint() (string, error) {
 	if instance.Status.DBAddress == "" {
 		return "", fmt.Errorf("external DBEndpoint not ready yet for %s", instance.Spec.DBType)
 	}
-	return instance.Status.DBAddress, nil
+	//return instance.Status.DBAddress, nil
+	//return "tcp:ovsdbserver-sb.openstack.svc:6642", nil
+	dns_hostname := "tcp:ovsdbserver-sb." + instance.Namespace + ".svc:6642"
```

Comment:
Yeah, I think it could be good to update DBAddres (with ovsdbserver-sb." + instance.Namespace + ".svc) and InternalDBAddress with "ovsdbserver-sb." + instance.Namespace + ".svc.cluster.local" (/me needs to look exactly how coreDNS sets DNS)

Reply:
By default, even when pulling in resolv.conf, it doesn't appear to append search domains.
```
>>> import unbound
>>> ctx = unbound.ub_ctx()
>>> ctx.resolvconf("/etc/resolv.conf")
0
>>> status, result = ctx.resolve("source")
>>> result.havedata
0
>>> status, result = ctx.resolve("source.redhat.com", unbound.RR_TYPE_A, unbound.RR_CLASS_IN)
>>> result.havedata
1
>>> result.data.address_list
['52.247.118.198']
```
As far as CNAMEs, that should be fine. `www.redhat.com` is a CNAME and:
```
>>> status, result = ctx.resolve("www.redhat.com", unbound.RR_TYPE_A)
>>> result.data.address_list
['96.16.244.143']
>>> status, result = ctx.resolve("www.redhat.com", unbound.RR_TYPE_AAAA)
>>> result.data.address_list
['38.0.20.4.100.0.22.150.0.0.0.0.0.0.13.68', '38.0.20.4.100.0.22.137.0.0.0.0.0.0.13.68']
```
It *may* be configurable via an unbound.conf to set a search domain, and we can pass those via an environment variable. But it seems like a good practice to use the FQDN.